Booting/linux-bootstrap-5.md

```diff
@@ -164,7 +164,7 @@ After it we get `.text` section address and jump to it:
 Last preparation before kernel decompression
 --------------------------------------------------------------------------------
 
-`.text` sections starts with the `relocated` label. For the start there is clearing of the `bss` section with:
+`.text` sections start with the `relocated` label. For the start there is clearing of the `bss` section with:
```

Comment:
Hi @ruthgrace,

There must be: '.text section' instead of '.text sections', can you please fix this and put back `starts`

Thank you